src/components/IndiaStateMap/Switch.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const Switch = ({ mode, setMode, activeAttribute }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="switch">
+        <button
+          className={
+            "btn btn-chart " + (mode === 'normal' ? activeAttribute : "")
+          }
+          onClick={() => { setMode('normal') }}
+        >
+          Normal
+      </button>
+        <button
+          className={
+            "btn btn-chart " + (mode === 'per million' ? activeAttribute : "")
+          }
+          onClick={() => { setMode('per million') }}
+        >
+          Per Million
+      </button>
+      </div>
+      <span style={{ fontSize: '0.8em', color: '#7d7d7d' }}>{t('Use the buttons to see the cases as per the state population in million')}</span>
```

Comment:
Would it be possible to add the population source here? Just a nit-pick, you can simply add a `[src]` at the end of this line and link it to the population data source.

Reply:
Also, can we change the text to 
```
Use the buttons to switch between Total Cases and Cases per million population
```

Reply:
Need more clarification

Reply:
I was suggesting something like this

```suggestion
      <span style={{ fontSize: '0.8em', color: '#7d7d7d' }}>{t('Use the buttons to switch between Total Cases and Cases per million population <a href="your_population_data_src">[Population Src]</a>')}</span>
```